ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
```

Comment:
I agree about lower case but why do not call simple otsig-id ?  OTSi-group in my view could be misleading: as you know theoretically OTSi can belong to different OTSiG and an otsi-group could be exactly that case.

Reply:
I think `otsig-id `would be the name of the key but the name of the list could be `otsig`. The only doubt with `otsig `is readability, expecially when we need to differentiate between `otsi `and `otsig`.

I do not understand "OTSi can belong to different OTSiG ": as far as I have understood, an OTSi belongs to one and only one OTSiG.

If not, the model is not correct (no matter on the naming) since the OTSis are listed within the OTSiG.

Reply:
> I think `otsig-id `would be the name of the key but the name of the list could be `otsig`. The only doubt with `otsig `is readability, expecially when we need to differentiate between `otsi `and `otsig`.
> 
> I do not understand "OTSi can belong to different OTSiG ": as far as I have understood, an OTSi belongs to one and only one OTSiG.
> 
> If not, the model is not correct (no matter on the naming) since the OTSis are listed within the OTSiG.
Yes, I was referring to NMCG not OTSiG , you're right. 
I think otsi-group is probably the better solution for naming. What about the key? otsi-group-id ?